tests/test-utils/makeRequestProcessor.ts

```diff
@@ -42,6 +48,17 @@ export default function makeRequestProcessor(name: string, userOpts?: MakeFuncti
             return next('route');
          } else if (opts.passesErrorToNext) {
             return next(err);
+         } else if (opts.returnsRejectedPromise) {
+            return new Promise((_resolve, reject) => setTimeout(() => reject(`Rejection from "${name}"`), 4));
```

Comment:
4ms is what we've been using in internal projects. Guess it seemed like a good value to @jthomerson at the time. I'm going with 42ms was too long.

Reply:
Yeah, it was just "make the test actually async" because I'd seen too many promise-related tests fall short because they tested promises, but actually ran synchronously. One ms would likely be just as good ... can't remember if there was more depth to it than that at the time 🤷‍♂️